vignettes/daily-series.Rmd

```diff
@@ -14,143 +14,140 @@ knitr::opts_chunk$set(
-
-This vingette describes the production processes of trendecon. The production function start with `proc_` and usually operate on the file system. That means that calling this functions will write to your disk.
-
+Once the combination of useful keywords is selected we might want to use them as economic indicators in production and display them on [trendecon.org](https://www.trendecon.org/). This vignette describes the steps for adding new indicators to the production processes of "Trendecon".
 
-## Directory structure
+## Directory structure ##
 
-The `proc_` functions write to the working directory. If you are unsure about your current working directory, use `getwd()`.
+The names of production functions within the `trendecon` package start with `proc_`.
+They usually operate on the file system which means that calling these functions will write files to disk.
+The `proc_` functions will save files within two folders: `data` and `raw`, which are located under the working directory (`getwd()`).
+`raw` folder holds all the data downloaded from Google, plus some transformations, such as seasonally adjusted or combined series.
+`data` folder, on the other hand, collects the final indicators - the data displayed on the website.
+If not present these folders will be created when calling `proc_keyword_init` for the first time.
 
-Within the working directory, the `proc_` functions assume the existence of two folders,
+Within these folders, every used `geo` location will have its own subfolder.
+Thus, in the end, if we used Swiss and Austrian indicators, we would end up with the following directory structure:
 
-- `data`
-- `raw`
-
-which, if not present, are created when calling `proc_keyword_init` for the first time.
-
-Within these folders, there is a subfolder for every `geo` location.
-Thus, you may see the following subfolders
-
-- `data/ch`
-- `data/at`
-- `raw/ch`
-- `raw/at`
-
-What is the difference between `data` and `raw`? In the `raw` folder, we collect all the data downloaded from Google, plus some transfromations, such as the seasonally adjusted or the combined series.
-
-In the `data` folder, we collect our final indicators - the data you see on the website.
+```
+│
+├── data
+│   ├── at
+│   └── ch
+│
+└── raw
+    ├── at
+    └── ch
+```
 
 
-## Initial Download
+## Initial download ##
 
-To include a new series, each keyword must be initiated. Be careful, as this causes a lot of queries to Google.
+To include a new series, each keyword must first be initiated.
+Our goal is to produce long daily time series, ideally from 2006.
+However, Google does not provide daily or weekly data for such a long time period.
+Hence we need to circumvent the problem by applying a moving window of daily and weekly queries over the whole time period.
+So be careful, as this causes a lot of queries to Google.
```

Comment:
This line made me insecure. You could add, why we should careful (blocking by the API?And yes, for how long?) Or could we delete it?

Reply:
I like the small addition!